src/screens/internal/OverridesOverview/OverridesOverview.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import jsonOverrides from '../../../cms-content/region-overrides/region-overrides.json';
+import { parseOverrides } from './utils';
+import { Box, Tooltip, Typography } from '@material-ui/core';
+
+const renderDataGridToolTip = (params: any) => (
+  <Tooltip title={params.value}>
+    <span>{params.value}</span>
+  </Tooltip>
+);
+
+function OverridesOverview() {
+  const regionOverriedes = parseOverrides(jsonOverrides.overrides);
+  // Filter out overrides that block a timeframe in the past.
+  const activeOverrides = regionOverriedes.filter(
+    override =>
+      override.endDate === null || new Date(override.endDate) > new Date(),
+  );
```

Comment:
FWIW- It might be nice to remove "metrics." from the metric name and format the dates as just "YYYY/MM/DD".  Right now they both get clipped (and for some reason the date doesn't show a tooltip).

![image](https://user-images.githubusercontent.com/206364/213331471-f3b419a5-d533-4046-8593-ec54ccfe4c8a.png)

Reply:
I'd also maybe reverse the list so the most recently-added overrides are on top.